src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -487,6 +487,17 @@ protected virtual string GetInsertionText(CompletionItem item, char ch)
         // This is used to decide which provider we'd collect target type completion telemetry from.
         protected virtual bool ShouldCollectTelemetryForTargetTypeCompletion => false;
 
+        protected static CompletionChange GetCompletionChangeWithParenthesis(
+            string insertionText, char commitKey, TextSpan itemSpan)
+        {
+            var text = string.Concat(insertionText + "()", commitKey);
```

Comment:
why `commitKey` is needed as papameter? It will always be ";". right?

Reply:
This is true for this PR. But I also have a plan to  also add '.' as the commit key. (Behave the same as ';')
But I am fine to change here to ';' to make this PR more clear